ios/fastlane/Changelog.txt

```diff
@@ -0,0 +1 @@
+Bugs fixing and performance improvements
```

Comment:
This CHANGELOG file can be updated to add custom information to external testers